client/my-sites/posts/post-controls.jsx

```diff
@@ -1,228 +0,0 @@
-/**
- * External dependencies
- */
-import React, { PropTypes } from 'react';
```

Comment:
@Copons do you mind rebasing to rename `client/my-sites/posts/post-controls.jsx` in one commit and apply changes to the components in follow ups commits? We'll lose component history otherwise for `client/my-sites/posts/post-controls.jsx` since it looks like this this is being inferred as a delete/add.